var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -29,7 +30,6 @@ class QuantumEspresso(Package):
     version('5.4',   '085f7e4de0952e266957bbc79563c54e')
     version('5.3',   'be3f8778e302cffb89258a5f936a7592')
     version('develop', branch='develop')
```

Comment:
Can you move `develop` to the top? We like to sort versions in descending order.